src/sys/windows/udp.rs

```diff
@@ -57,31 +57,39 @@ impl UdpSocket {
     }
 
     pub fn send_to(&self, buf: &[u8], target: SocketAddr) -> io::Result<usize> {
-        try_io!(self, send_to, buf, target)
+        try_io!(self.send_to(buf, target), |_| true)
     }
 
     pub fn recv_from(&self, buf: &mut [u8]) -> io::Result<(usize, SocketAddr)> {
-        try_io!(self, recv_from, buf)
+        try_io!(self.recv_from(buf), |_| true)
     }
 
     pub fn peek_from(&self, buf: &mut [u8]) -> io::Result<(usize, SocketAddr)> {
         try_io!(self, peek_from, buf)
     }
 
     pub fn send(&self, buf: &[u8]) -> io::Result<usize> {
-        try_io!(self, send, buf)
+        try_io!(self.send(buf), |_| true)
     }
 
     pub fn recv(&self, buf: &mut [u8]) -> io::Result<usize> {
-        try_io!(self, recv, buf)
+        try_io!(self.recv(buf), |_| true)
     }
 
     pub fn peek(&self, buf: &mut [u8]) -> io::Result<usize> {
         try_io!(self, peek, buf)
     }
 
     pub fn connect(&self, addr: SocketAddr) -> io::Result<()> {
-        self.inner.connect(addr)
+        let result = self.inner.connect(addr);
```

Comment:
Are you sure this is required?

Reply:
Don't think its required, see comment in `udp_socket_edge_triggered_behavior` test.

Reply:
you're right, not required.